lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -1,109 +1,197 @@
+  nxapi:
+    config_get_token_append: '/^bestpath med non-deterministic$/'
+    config_set_append: '<state> bestpath med non-deterministic'
+    default_value: false
 
 cluster_id:
-  config_get_token_append: '/^cluster-id (\S+)$/'
-  config_set_append: '<state> cluster-id <id>'
-  default_value: ""
+  grpc:
+    config_get_token_append: '/^bgp cluster-id (\S+)$/'
+    config_set_append: ' <state> bgp cluster-id <id>'
+    default_value: ""
+  nxapi:
+    config_get_token_append: '/^cluster-id (\S+)$/'
+    config_set_append: '<state> cluster-id <id>'
+    default_value: ""
 
 confederation_id:
-  config_get_token_append: '/^confederation identifier (\d+.?\d+?)$/'
-  config_set_append: '<state> confederation identifier <id>'
-  default_value: ""
+  grpc:
+    config_get_token_append: '/^bgp confederation identifier (\d+.?\d+?)$/'
+    config_set_append: ' <state> bgp confederation identifier <id>'
+    default_value: ""
+  nxapi:
+    config_get_token_append: '/^confederation identifier (\d+.?\d+?)$/'
+    config_set_append: '<state> confederation identifier <id>'
+    default_value: ""
 
+#TODO regexp is broken
 confederation_peers:
-  config_get_token_append: '/^confederation peers (.*)$/'
-  config_set_append: '<state> confederation peers <peer_list>'
-  default_value: ""
+  grpc:
+    config_get_token_append: '/^bgp confederation peers (.*)$/'
+    config_set_append: ' <state> bgp confederation peers <peer_list>'
+    default_value: ""
+  nxapi:
+    config_get_token_append: '/^confederation peers (.*)$/'
+    config_set_append: '<state> confederation peers <peer_list>'
+    default_value: ""
 
 create_destroy_neighbor:
   config_set_append: '<state> neighbor <nbr>'
 
 enforce_first_as:
-  config_get_token_append: '/^enforce-first-as$/'
-  config_set_append: '<state> enforce-first-as'
-  default_value: true
+  grpc:
+    config_get_token_append: '/^bgp enforce-first-as disable$/'
+    config_set_append: ' <state> bgp enforce-first-as disable'
+    default_value: true
+  nxapi:
+    config_get_token_append: '/^enforce-first-as$/'
+    config_set_append: '<state> enforce-first-as'
+    default_value: true
 
 feature:
-  config_get: "show running bgp"
-  config_get_token: '/^feature bgp$/'
-  config_set: "<state> feature bgp"
+  grpc:
+    config_get: "show running-config router bgp"
+    config_get_token: '/^router bgp$/'
+    config_set: "<state> router bgp"
+  nxapi:
+    config_get: "show running bgp all"
+    config_get_token: '/^feature bgp$/'
+    config_set: "<state> feature bgp"
 
 graceful_restart:
-  config_get_token_append: '/^graceful-restart$/'
-  config_set_append: '<state> graceful-restart'
-  default_value: true
+  grpc:
+    config_get_token_append: '/^bgp graceful-restart$/'
+    config_set_append: ' <state> bgp graceful-restart'
+    default_value: true
+  nxapi:
+    config_get_token_append: '/^graceful-restart$/'
+    config_set_append: '<state> graceful-restart'
+    default_value: true
 
+#Does not exist in IOS XR
```

Comment:
Can you mark this as a 'TODO:'? This is one that will need to be updated once we have proper `exclude_platform` capability in the YAML.